server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+
+const pr = require('./posts-router');
```

Comment:
```suggestion
const postsRouter = require('./posts-router');
```
Please make your variable names more semantic. I know it's easier to type pr but it's a lot less easy for another developer to come in and immediately understand what `pr` means.